ingestion/src/metadata/cmd.py

```diff
@@ -210,11 +210,17 @@ def backup_args(parser: argparse.ArgumentParser):
         default=None,
     )
     parser.add_argument(
-        "--upload",
+        "--upload-s3",
         help="S3 endpoint, bucket & key to upload the backup file",
         nargs=3,
         default=None,
     )
+    parser.add_argument(
+        "--upload-azure",
+        help="Account-url, Container, Tenant-id, Client-id & Client-secret to upload the backup file",
+        nargs=5,
+        default=None,
```

Comment:
when we try to upload the backup file to s3 we pickup the creds from like access key and key id from environment (basically user has to setup aws cli) can you check if a similar solution would work here instead of taking everything as argument

Reply:
Done.